### PR TITLE
Enes JavaScript3 Week1 Homework

### PR DESCRIPTION
**Pull request by eneserdek**

### CODE REVIEW

File: homework/index.js [hunk 1]
```diff
@@ -29,19 +29,127 @@
     });
     return elem;
   }
-
+  function singlePageApplication(arr) {
+    //header
```
Comment (paulbremer) [inline homework/index.js]: If you run `npm run lint` in your terminal you'll see 19 errors, so I'm not gonna point out all the errors again. Please check your `npm run lint` and take a look at the errors, most of them are really easy to solve. Next time check your PR if all checks are passing!

File: homework/index.js [hunk 2]
```diff
@@ -29,19 +29,127 @@
...
+    const rightDiv = createAndAppend('div', containerDiv, {
+      id: 'right-div',
+      class: 'contained',
+    });
+    const tHeads = ['Repository: ', 'Description: ', 'Forks: ', 'Update: '];
+    const table = createAndAppend('table', rightDiv, {
+      id: 'tableOfInformation',
+    });
+    for (let i = 0; i < tHeads.length; i++) {
+      var firstTime = ['name', 'description', 'forks', 'updated_at'];
```
Comment (paulbremer) [inline homework/index.js]: Why are you using a `var`? Do you know a better way to solve this using `const`?

File: homework/index.js [hunk 3]
```diff
@@ -29,19 +29,127 @@
...
+          text: arr[0][firstTime[i]],
+          id: `tableData.${i}`,
+        });
+      }
+    }
+    selectMenu.addEventListener('change', event => {
+      for (let i = 0; i < 4; i++) {
+        const changed = document.getElementById(`tableData.${i}`);
+        if (i === 0) {
+          changed.innerHTML = `<a target = "_blank" href = ${arr[event.target.value].html_url}> ${
```
Comment (paulbremer) [inline homework/index.js]: Syntax with all the spaces looks weird and might break things, it should be like this; `target="_blank" href="${}"`

File: homework/style.css [hunk 4]
```diff
@@ -1,3 +1,50 @@
...
+  align-content: center;
+}
+
+h1 {
+  color: white;
+  margin: 0 20px 0 20px;
+}
+
+.container {
+  display: flex;
```
Comment (paulbremer) [inline homework/style.css]: Nice use of `flex`! But using media queries you could make it even more responsive!

**Comment by wouterkleijn** [comment]
Thanks, @paulbremer for your comments. @eneserdek, can you have a look and implement the changes - and ask for help if you need it!

**Comment by paulbremer** [comment]
@eneserdek changes are looking good! 👍

**Pull request closed without merging**